### DISCUSSION TITLE
How to install a GitHub release?

### DISCUSSION THREAD

**Discussion by a1russell** [marlonrichert/zsh-snap#172] [Q&A] [answered]
How can I install something that is published as a GitHub release, rather than cloning the entire git repository?

For example, I'd like to manage [Starship](https://github.com/starship/starship) entirely in my .zshrc without having to use another package manager to install starship first. I would love it if I could have it downloaded in parallel with my `zinit clone` step to take advantage of the speed benefits that brings me, without manually writing a block of logic using curl. Is this currently possible?

**Comment by marlonrichert** [comment, marked as answer]
Releases are not kept in a repo, but are simply binary downloads on GitHub. Ergo, it’s not possible to use `znap clone` for this.

**Reply by a1russell** [reply]
Apologies if my question was not clear. I did not think that `znap clone` could be used for GitHub releases. Rather, I was asking if there _is_ an equivalent command that could be used for GitHub releases, or any other downloadable binary. Based on your answer, I'm assuming not, though. I suppose I'll continue using `curl`.

**Reply by marlonrichert** [reply]
No, there currently isn’t. Feel free to submit a feature request, though. 🙂
